src/lib.rs

```diff
@@ -55,7 +55,7 @@
 //! to the network through any node, and exchange public keys with it. That node becomes a
 //! bootstrap node for the client, and messages to and from the client will be routed over it.
 //!
-//! ```
+//! ```no_run
```

Comment:
Do these really break with mock Crust? After all, the signature of `Client::new` didn't change ...?